spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
+    let!(:articles) { create_list(:article, 10) }
+    before { get api_v1_articles_path, headers: headers }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns the list of articles' do
+      expect(json.map { |x| x['id'] }).to eq [*1..10]
+    end
+  end
+
+  describe 'GET #show' do
+    let!(:articles) { create_list(:article, 5) }
+    context 'existing article' do
+      before { get '/api/v1/articles/3', headers: headers }
+
+      it 'returns a successful response' do
+        expect(response).to be_successful
+      end
+
+      it 'returns existing article with specified id' do
+        expect(json['id']).to eq 3
+      end
+    end
+
+    context 'not existing article' do
+      it 'raises an error' do
+        assert_raises(ActiveRecord::RecordNotFound) do
+          get '/api/v1/articles/1234', headers: headers
+        end
+      end
+    end
+  end
+
+  describe 'GET #create' do
+    it 'returns a successful response' do
+      get api_v1_articles_path,
+          params: create(:article),
+          headers: headers
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'GET #update' do
+    it 'returns a successful response' do
+      get "/api/v1/articles/#{create(:article).id}", headers: headers
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'POST #create' do # rubocop:disable Metrics/BlockLength
+    context 'with valid attributes' do
+      let!(:category) { create(:category) }
+
+      it 'creates the record in the database' do
+        expect do
+          post api_v1_articles_path,
+               params: { article: { title: 'Title-1-', text: 'Test', category_id: category.id } },
+               headers: headers
+        end.to change(Article, :count).by(1)
```

Comment:
I would test attributes too cause it is a risk some attribute will be filtered in your params